tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -192,14 +192,14 @@ SectionGraph."PIPE_CAP_VIRT" {
 DAI_ADD(sof/pipe-dai-playback.m4,
```

Comment:
@mrajwa care to be more explicit with the commit message. So, as far as i understand the pipelines were interrupt driven but something was wrong with that and we want them to be timer driven?

The commit message should say why the old code isn't OK and why do we need to change it.